lib/rubocop/cop/style/format_string.rb

```diff
@@ -40,6 +40,50 @@ def message(detected_style)
         def method_name(style_name)
           style_name == :percent ? 'String#%' : style_name
         end
+
+        def autocorrect(node)
+          lambda do |corrector|
+            _, method = *node
+            if method == :%
+              autocorrect_from_percent(corrector, node)
+            elsif style == :percent
+              autocorrect_to_percent(corrector, node)
+            else
+              corrector.replace(node.loc.selector, style.to_s)
+            end
+          end
+        end
+
+        private
+
+        def autocorrect_from_percent(corrector, node)
+          string, *_, args = *node
```

Comment:
This local var assigns a format string like `"%d"` :)

I think about `format` or `placeholder` might be better . c4542edafe7c6a84